compiler/noirc_frontend/src/hir/def_collector/dc_mod.rs

```diff
@@ -679,14 +683,6 @@ impl<'a> ModCollector<'a> {
         is_contract: bool,
     ) -> Result<ModuleId, DefCollectorErrorKind> {
         let parent = Some(self.module_id);
-
-        // Note: the difference between `location` and `mod_location` is:
-        // - `mod_location` will point to either the token "foo" in `mod foo { ... }`
-        //   if it's an inline module, or the first char of a the file if it's an external module.
-        // - `location` will always point to the token "foo" in `mod foo` regardless of whether
-        //   it's inline or external.
-        // Eventually the location put in `ModuleData` is used for codelenses about `contract`s,
-        // so we keep using `location` so that it continues to work as usual.
         let location = Location::new(mod_name.span(), mod_location.file);
```

Comment:
```suggestion
        // Note: the difference between `location` and `mod_location` is:
        // - `mod_location` will point to either the token "foo" in `mod foo { ... }`
        //   if it's an inline module, or the first char of a the file if it's an external module.
        // - `location` will always point to the token "foo" in `mod foo` regardless of whether
        //   it's inline or external.
        // Eventually the location put in `ModuleData` is used for codelenses about `contract`s,
        // so we keep using `location` so that it continues to work as usual.
        let location = Location::new(mod_name.span(), mod_location.file);
```